src/Http/ServerRequest.php

```diff
@@ -534,24 +534,24 @@ public function clearDetectorCache(): void
      * @param array $args Array of custom detector arguments.
      * @return bool Whether the request is the type you are checking.
      */
-    protected function _is(string $type, array $args): bool
+    protected function isType(string $type, array $args): bool
```

Comment:
`is` already exists, therefore I thought this would be a good name for this sub-method